english/faris-in-wonderland/the-giant.md

```diff
@@ -0,0 +1,6 @@
+once upon time , there was a boy who love discover and fun around
+and his name was faris , one day while he was playing in front yard
+he found a little bottel with some mystery language label ,
+he decided to drink this bottel and suddenly he close his eyes
+and found him self in a wondrland.
+[Back to English](https://github.com/udacity/create-your-own-adventure/tree/master/english)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.